lib/compat.php

```diff
@@ -204,7 +204,16 @@ function gutenberg_render_block_with_assigned_block_context( $pre_render, $parse
 
 	/** This filter is documented in src/wp-includes/blocks.php */
 	$parsed_block = apply_filters( 'render_block_data', $parsed_block, $source_block );
-	$context      = array( 'postId' => $post->ID );
+	$context      = array(
+		'postId'   => $post->ID,
+		/*
+		 * The `postType` context is largely unnecessary server-side, since the
+		 * ID is usually sufficient on its own. That being said, since a block
+		 * manifests is expected to be shared between the server and the client,
```

Comment:
```suggestion
		 * ID is usually sufficient on its own. That being said, since a block's
		 * manifest is expected to be shared between the server and the client,
```

Reply:
Good catch. I was playing with the phrasing and it often results in some strange grammar 😅